compiler/lexer.cpp

```diff
@@ -163,9 +181,44 @@ plungefile(char* name, int try_currentpath, int try_includepaths)
             result = plungequalifiedfile(path);
         }
     }
+
+    if(pcwd != NULL) {

```

Comment:
`if (!pwd) {`

Reply:
I don't think the "if" and parens nuzzling was changed, but everything else looks good to me.  Thanks for doing this and adding tests.

Reply:
i don't really get what you mean with that, i could be missing something but i think i got everything cleaned up.